Files/MultilingualResources/Files.de-DE.xlf

```diff
@@ -575,7 +575,11 @@
           <source>MD5Hash:</source>
           <target state="translated">MD5Hash:</target>
         </trans-unit>
+        <trans-unit id="SettingsPageHeader.Text" translate="yes" xml:space="preserve">
```

Comment:
There is already exists translated string Settings under id "SettingsNav.PaneTitle", so you can change name to "SettingsPageHeader.Text" and no need to translate it again